examples/guard_riverpod/README.md

```diff
@@ -0,0 +1,19 @@
+# Guard Riverpod Example
+
+Example using [riverpod](https://pub.dev/packages/riverpod) package, with the release
+of `1.0.0-dev.7`, which drastically changes how things are done with this package
+(if comparing to `0.14.0` or previous versions).
```

Comment:
I think this is unnecessary as `BeamGuard`s have been pretty much the same since their inception.